### DISCUSSION TITLE
Watched time not synced

### DISCUSSION THREAD

**Discussion by pargnyce** [Taxel/PlexTraktSync#1741] [Support]
Hi. I'm using plextraktsync via Windows. I noticed that when I sync, the watched time is not equal to trakt. For example, the watched time in my server is 12am, but in trakt, it is always -8 hours (4pm). Is there a way on how to set timezone in plextraktsync config?

https://github.com/Taxel/PlexTraktSync/issues/1765

**Comment by glensc** [comment]
If your time and timezone are correct in related places (plex server, place where plextraktsync runs), then it works as expected. timezones can be converted as needed.

Otherwise, not much information to go on here. You should rather open bug report issue and provide details of your system.

**Reply by pargnyce** [reply]
I run it on the same machine (plex server and plextraktsync)

**Reply by glensc** [reply]
well then. everything should just work. 

you need to create bug report issue with more details (especially your setup and test inspect runs) if you want somebody else look at the problem.

**Reply by glensc** [reply]
user doesn't listen. i told to create issue and link. closing
